source/_integrations/airnow.markdown

```diff
@@ -24,6 +24,20 @@ To generate an AirNow API key, go to the [AirNow Developer Tools Page](https://d
 
 {% include integrations/config_flow.md %}
 
+## Troubleshooting
+
+The EPA AirNow API is often flaky and will occasionally not return any results for a particular location.  This will
+prevent the integration from being added to Home Assistant, but the situation is usually temporary and will resolve
+itself later.
```

Comment:
```suggestion
The EPA AirNow API is often flaky and will occasionally not return any results for a particular location. This will prevent the integration from being added to Home Assistant, but the situation is usually temporary and will resolve itself later.
```